specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Makes sense; we just need to figure out the best way to handle it. We also need to figure out how we're getting election definitions into the system, but that's a whole other thing.